internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -149,41 +148,14 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 		options.NewClient = client.New
 	}
 
-	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
-	log = log.WithValues("Namespace", namespace)
-	var watchNamespaces []string
-	if found {
-		log.V(1).Info(fmt.Sprintf("Setting namespace with value in %s", k8sutil.WatchNamespaceEnvVar))
-		if namespace == metav1.NamespaceAll {
-			log.Info("Watching all namespaces.")
-			watchNamespaces = []string{metav1.NamespaceAll}
-		} else {
-			if strings.Contains(namespace, ",") {
-				log.Info("Watching multiple namespaces.")
-				watchNamespaces = strings.Split(namespace, ",")
-			} else {
-				log.Info("Watching single namespace.")
-				watchNamespaces = []string{namespace}
-			}
-		}
-		// TODO: remove this when loading from config file option is removed
-	} else if options.Cache.Namespaces == nil { //nolint:staticcheck
-		log.Info(fmt.Sprintf("Watch namespaces not configured by environment variable %s or file. "+
-			"Watching all namespaces.", k8sutil.WatchNamespaceEnvVar))
-		watchNamespaces = []string{metav1.NamespaceAll}
-	}
+	namespaces := configureWatchNamespaces(&options, log)
 
 	err = setAnsibleEnvVars(f)
 	if err != nil {
 		log.Error(err, "Failed to set environment variable.")
 		os.Exit(1)
 	}
 
-	options.NewCache = func(config *rest.Config, opts cache.Options) (cache.Cache, error) {
```

Comment:
I think this is still needed in order to properly create the new manager. (The e2e tests are failing because they're unable to create a new manager.)

Reply:
I think it should be fine as is because instead of using a `NewCache` function we are configuring the cache configurations in https://github.com/operator-framework/ansible-operator-plugins/pull/51/files#diff-8f8a3e7e238207f04c7258c16670966e1e72ca063527fcd0ec6429c475d35cd0R356

Reply:
Like @everettraven the configruation said, the cache has moved. This is the same as the `helm plugin`

The E2E is failing because the proxy can't find the namespaces, the proxy code is different then a typical manager code (and uses different custom datatypes for this project), that I don't fully understand what/why this conversion is happening.

Reply:
@acornett21 I can try to take a deeper look at this sometime later today

Reply:
@everettraven I was able to get this setup on a VM, and have made complete changes now.